jsapp/js/account/security/password/updatePasswordForm.component.tsx

```diff
@@ -105,7 +105,10 @@ export default function UpdatePasswordForm(props: UpdatePasswordFormProps) {
         }
 
         setIsPending(false);
-        notify(t('failed to change password'), 'error');
+        // Accounting for double error edge case when error status is 403
+        if (errorObj.status !== 403){
+          notify(t('failed to change password'), 'error');
+        }
```

Comment:
This is a good first attempt! Still, if we get a 500 error, it'll still show a double notification. You might want to take a look at `fetchPatch()`  (and `FetchDataOptions`).